src/test/kotlin/org/rust/ide/inspections/RsBorrowCheckerInspectionTest.kt

```diff
@@ -125,6 +125,16 @@ class RsBorrowCheckerInspectionTest : RsInspectionsTestBase(RsBorrowCheckerInspe
         }
     """)
 
+    fun `test no highlight for mutable for loops`() = checkByText("""
+        fn test() {
+            let mut xs: Vec<Vec<usize>> = vec![vec![1, 2], vec![3, 4]];
+
+            for test in &mut xs {
+                <error>test</error>.push(0)
```

Comment:
Hm, I think there shouldn't be an error here? At lest the following compiles fine:

```Rust
fn main() {
    let mut xs: Vec<Vec<usize>> = vec![vec![1, 2], vec![3, 4]];
    for test in &mut xs {
        test.push(0)
    }
}
```

Reply:
If this is a test for the current behavior which should be fixed later, could you add a `TODO: this should not show error here` ?

Reply:
I add a note